### PR TITLE
[no-jira] Use Actions, not Travis; also, apply `standard`

### PR DESCRIPTION
**Pull request by knutsenm**
Once deployed, I must

- [x] Disable the Travis build
- [x] `git tag v1.7.1 master; git push origin v1.7.1`
- [x] Make `lint-and-test` required in branch protection rules

### CODE REVIEW

File: lib/global_registry.rb [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
-Dir[File.dirname(__FILE__) + '/global_registry/*.rb'].each do |file|
+Dir[File.dirname(__FILE__) + "/global_registry/*.rb"].each do |file|
   require file
 end
 
 module GlobalRegistry
   class << self
-    attr_accessor :base_url, :access_token
+    attr_accessor :access_token
+    attr_writer :base_url
```
Comment (knutsenm) [inline lib/global_registry.rb:12]: Does this look ok? Rubocop complained,
```
Lint/DuplicateMethods: Method `GlobalRegistry.base_url` is defined at both lib/global_registry.rb:11 and lib/global_registry.rb:17
```

**Comment by knutsenm** [comment]
@jbirdjavi should I bump the gem version? Should there be tags for that, and for v1.7.0?

**Comment by knutsenm** [comment]
Also, should there be a Gemfile.lock? Otherwise, every Action is going to run the very latest `standard`

File: lib/global_registry.rb (hunk 1 above)
Comment (jbirdjavi) [reply lib/global_registry.rb:12]: Yeah that should be fine.

**Review by jbirdjavi — DISMISSED** [review]
Thanks!

Yeah I'd bump the version (probably patch level is fine). I always forget about tagging these, but feel free to do that too.

**Review by jbirdjavi — APPROVED** [review]

**Pull request merged**